### PR TITLE
Streamlit Tutorial: How to use streamlit to deploy your app for your hackathon project

### PR DESCRIPTION
**Pull request by Mohammad-Noor-debug**
New tutorial

### CODE REVIEW

**Comment by Mohammad-Noor-debug** [comment]
https://drive.google.com/drive/folders/1-tyLtqoJP3PU2ug6vFLw5yGQkPb5BKSm?usp=drive_link


Here is the video tutorials as well

**Comment by ezzcodeezzlife** [comment]
please follow the exact markdown structure of previous tutorials, thanks!

**Comment by Mohammad-Noor-debug** [comment]
> please follow the exact markdown structure of previous tutorials, thanks!

Can you please guide me through it? I couldn’t find it at all

**Comment by septiannugraha** [comment]
You can refer to [this link](https://github.com/lablab-ai/community-content/blob/main/tutorials/README.md) as guidance.

**Comment by Mohammad-Noor-debug** [comment]
I believe the only thing I’ve done wrong is the image path file, correct?

**Comment by septiannugraha** [comment]
> I believe the only thing I’ve done wrong is the image path file, correct?

I'm sorry. To be fair, the usage of MDX's  tag is the only thing you did right in the tutorial.

**Comment by Mohammad-Noor-debug** [comment]
Oh, could you please guide me through it then? It’s my first time doing a tutorial, sorry

**Comment by septiannugraha** [comment]
> Oh, could you please guide me through it then? It’s my first time doing a tutorial, sorry

You can refer to earlier tutorials such as [this one}(https://github.com/lablab-ai/community-content/pull/224#issuecomment-1627379778) if the explanation isn't clear. Basically here's the requirements:

- use table at the start of the mdx file
```mdx
---
title: 'Title of the tutorial'
description: 'A Longer Description of the Tutorial'
authorUsername: 'author-name'
---
```

- Use headings (commonly notated by hash (#) symbol, if you're not aware of this, you really should [learn markdown](https://www.markdownguide.org/) first. Basically, the numbers of hash symbols indicate the level of the headings. e.g: # for heading 1, ## for heading 2, etc..)
- Use code blocks to indicate sections of code. This allows readers to conveniently copy the code for their own practice project without selecting other contents by accident. e.g:
```python
print('Hello world')
```
- Use MDX's  tag for images, make sure you've uploaded the image somewhere first. A popular choice would be [freeimagehost](freeimage.host). I believe you've done this one already.

Bottom line, you should learn markdown first if you didn't have firm enough grasps of some of the concepts.

**Comment by septiannugraha** [comment]
> Alright thank you so much, Could you possibly give me about two days to get this done? Thank you again

I'm not the reviewer though. You'd better ask @ezzcodeezzlife. You might be able to take your time as long as the issue's open.

**Comment by ezzcodeezzlife** [comment]
Hey, yes sure take your time. Just make sure the format & content is on point ;) 

have fun!

**Comment by ezzcodeezzlife** [comment]
@Mohammad-Noor-debug please update the format ❤️

**Comment by Mohammad-Noor-debug** [comment]
> @Mohammad-Noor-debug please update the format ❤️

Hey, sorry for not updating the format, I was a little busy, I'll try my best to update it later tonight

**Comment by ezzcodeezzlife** [comment]
> > @Mohammad-Noor-debug please update the format ❤️
> 
> Hey, sorry for not updating the format, I was a little busy, I'll try my best to update it later tonight

Hey, just wanted to ask whats the staus of this? Thanks! 🚀
